mitodl/ocw-studio
# Update eslint-config-mitodl

#### Pre-Flight checklist



#### What are the relevant tickets?
https://github.com/mitodl/eslint-config/pull/16, https://github.com/mitodl/eslint-config/pull/15

#### What's this PR do?
Updates to a new version of `eslint-config-mitodl`: https://github.com/mitodl/eslint-config/pull/16

The new config:
- supports Typescript, so Studio doesn't need to do that itself (hence smaller studio config)
- allows using Jest instead of Mocha... *Previously we were using Mocha linting rules in OCW Studio. But Studio doesn't use Mocha. Mocha and Jest are similar enough this didn't cause very many problems, but it does cause a few. E.g, with top-level hooks.*

#### How should this be manually tested?
1. Check that linting / formatting pass. (Or don't, since CI does.)
2. Introduce a linting error and check that linting fails.
